extensions/amp-google-assistant-assistjs/0.1/libs/compile.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+# install google closure library:  `npm install google-closure-library` at amphtml
+# folder
+
+# use closure builder to generate dependencies
+# BUILDER=../../../../node_modules/google-closure-library/closure/bin/build/closurebuilder.py
+# $BUILDER \
+#   --root=closure-library/ \
+#   --root=wrapper/ \
+#   --namespace="assistjs"
+
+CLOSURE_LIB=../../../../node_modules/google-closure-library
+
+npx google-closure-compiler \
+  --compilation_level ADVANCED_OPTIMIZATIONS \
+  --manage_closure_dependencies \
+  --define "goog.DEBUG=true" \
```

Comment:
Is debug needed? Setting it to false saves us 2kb brotli compressed.

Reply:
Yes correct. Removed. Will push it through the next commit.